src/horus_api.c

```diff
@@ -348,8 +348,11 @@ int extract_horus_binary(struct horus *hstates, char hex_out[], int uw_loc) {
     return hstates->crc_ok;
 }
 
-
 int horus_rx(struct horus *hstates, char ascii_out[], short demod_in[]) {
+    return horus_rx_iq(hstates, ascii_out, demod_in, 0);
+}
+
+int horus_rx_iq(struct horus *hstates, char ascii_out[], short demod_in[], int quadrature) {
```

Comment:
Pls use the convention ```_comp``` rather than ```_iq```, to be consistent with the rest of codec 2.